job-service-container/pom.xml

```diff
@@ -556,6 +556,35 @@
                                 </wait>
                             </run>
                         </image>
+
+                        <!-- Run the Job Service Scheduled Executor -->
+                        <image>
+                            <alias>job-service-scheduled-executor</alias>
+                            <name>${dockerJobServiceOrg}job-service-scheduled-executor${dockerProjectVersion}</name>
+                            <run>
```

Comment:
You should probably have a `wait` section in here also.

Reply:
Added